src/ni/vsbuild/BuildConfiguration.groovy

```diff
@@ -77,4 +79,31 @@ class BuildConfiguration implements Serializable {
 
       return list
    }
+
+   private static def replaceTags(def script, def jsonItem, def lvVersion) {
+      if(jsonItem instanceof java.lang.String ||
+         jsonItem instanceof org.codehaus.groovy.runtime.GStringImpl) {
+
+         def replacedValue = jsonItem
+         script.versionReplacementExpressions().each {expression ->
+            replacedValue = replacedValue.replaceAll("\\{$expression\\}", lvVersion)
+         }
+
+         return replacedValue
```

Comment:
Why the early return here but not elsewhere?

Reply:
This is the return from the recursive call. The other cases recursively call back into this method, so they don't return anything.